tests/core/editor/editor.js

```diff
@@ -377,5 +377,42 @@ bender.test( {
 		assert.areSame( 'foo', insertHtml.firstCall.args[ 0 ], 'insertHtml dataValue' );
 		assert.areSame( 'html', insertHtml.firstCall.args[ 1 ], 'insertHtml mode' );
 		assert.areSame( range, insertHtml.firstCall.args[ 2 ], 'insertHtml range' );
+	},
+
+	'test extraPlugins allows whitespaces': function() {
```

Comment:
There is `tests/core/config` directory – maybe it would be better/more readable to move these unit tests to dedicated file (`plugins.js`) in this directory?

Reply:
I'll also add ticket references to tests.